examples/stencil/README.md

```diff
@@ -0,0 +1,146 @@
+# [Stencil](https://stenciljs.com)
+
+For a new Stencil project follow the [getting started](https://stenciljs.com/docs/getting-started) guide.
+
+To add Tailwind to a Stencil project, start by installing `@fullhuman/postcss-purgecss`, `@stencil/postcss`, `autoprefixer`, `cssnano` and `tailwindcss`:
+
+```sh
+npm install @fullhuman/postcss-purgecss @stencil/postcss autoprefixer cssnano tailwindcss --save-dev
+```
+
+Next, set up the PostCSS plugin in stencil.config.ts
+
+```js
+import { Config } from '@stencil/core';
+import { postcss } from '@stencil/postcss';
+import autoprefixer from 'autoprefixer';
+
+const purgecss = require('@fullhuman/postcss-purgecss')({
+  content: ['./src/**/*.tsx', './src/index.html'],
+  defaultExtractor: content => content.match(/[A-Za-z0-9-_:/]+/g) || []
+});
+
+export const config: Config = {
+  globalStyle: 'src/global/app.css',
+  outputTargets: [
+    {
+      type: 'www',
+      serviceWorker: null,
+      baseUrl: 'http://localhost:5000'
+    }
+  ],
+  plugins: [
+    postcss({
+      plugins: [
+        require('tailwindcss')('./tailwind.config.js'),
+        autoprefixer(),
+        ...(process.env.NODE_ENV === 'production'
+          ? [purgecss, require('cssnano')]
+          : [])
+      ]
+    })
+  ]
+};
+```
+
+You only want to use purgecss for production builds so modify package.json to ensure builds are run the NODE_ENV set to production
+
+```js
+  "scripts": {
+    "build": "NODE_ENV=production stencil build --prerender",
+    "start": "stencil build --dev --watch --serve"
+  },
+```
+
+Consider using `cross-env` if your team uses multiple operating systems.
+
+Next, create a CSS file in your `global` folder for your Tailwind styles. We've used `src/global/css/app.css` for this example:
```

Comment:
Is `/css/` a typo?

Reply:
Yes - it should be `src/global/app.css` as per the configured `globalStyle` on [line 24](https://github.com/tailwindcss/setup-examples/pull/24/files#diff-f20b1de2196ecdeffaa392b4af09ee4fR24).

Reply:
I've updated the example